src/Http/Controllers/WysiwygControlller.php

```diff
@@ -30,4 +30,21 @@ public function getTwitterEmbedCode(Request $request): string
             'chrome'      => 'nofooter',
         ])->json()['html']);
     }
+
+    public function uploadImage(Request $request): array
+    {
+        $this->validate($request, [
+            'image' => 'required|image',
+        ]);
+
+        $file = $request->file('image');
+
+        $path = $file->storePubliclyAs(
+            'wysiwyg',
```

Comment:
can you make this folder configurable? it's not really `wysiwyg` and will differ between ark.io and later MSQ for example